tables/tests/test_vlarray.py

```diff
@@ -1359,7 +1359,7 @@ def test06a_Object(self):
 
         vlarray = self.h5file.create_vlarray(
             '/', "Object", atom=ObjectAtom())
-        vlarray.append([[1, 2, 3], "aaa", u"aaa���"])
+        vlarray.append([[1, 2, 3], "aaa", "aaa���"])
```

Comment:
These are interesting - the file has the coding cookie - `# -*- coding: latin-1 -*-` and if I reload the file using the Latin-1 encoding in VS Code, these bytes are decoded correctly.

The surprising thing is that GitHub is able to decode the bytes before this change was made and I'm not sure how. If I load the file on master with encoding set to UTF-8. I just see garbage - not the accented c-like characters that GitHub is displaying.

Reply:
https://github.com/PyTables/PyTables/commit/8bbe84f91cf81e4228e68a4ffa7084592e3a3861

Reply:
Ah, yes.  I forgot that the internal supported coding for `VLArray` is just `latin-1`.  So better don't change anything more here.

Reply:
i'm not exactly sure what you mean by that @FrancescAlted . Do you mean not remove the coding cookie in the file?

Reply:
Yes, I mean exactly that.  It is a test file, so I don't think this would be problematic.